src/game/Objects/Player.cpp

```diff
@@ -2854,7 +2854,8 @@ void Player::GiveLevel(uint32 level)
     data << uint32((int32(classInfo.basehealth) - int32(GetCreateHealth()))
         + ((int32(info.stats[STAT_STAMINA]) - GetCreateStat(STAT_STAMINA)) * 10));
     // for(int i = 0; i < MAX_POWERS; ++i)                  // Powers loop (0-6)
-    data << uint32(int32(classInfo.basemana)   - int32(GetCreateMana()));
+    data << uint32((int32(classInfo.basemana)   - int32(GetCreateMana()))
+        + ((int32(info.stats[STAT_INTELLECT]) - GetCreateStat(STAT_INTELLECT)) * 15));
```

Comment:
this should take into account the breakpoint at 20 int... the HP one probably should as well but i think almost every class/race combo starts with more than 20 stam

Reply:
Done.
Also fixed it showing mana gained on non mana classes.